### PR TITLE
alembic/ddl/sqlite.py: enhance the judgment condition to fix the problem of AttributeError: 'NoneType' object has no attribute 'persisted' in the requires_recreate_in_batch function.

### PR DESCRIPTION
**Pull request by hustliyilin**
Signed-off-by: YiLin.Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

File: alembic/ddl/sqlite.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def requires_recreate_in_batch(
                     return True
                 elif (
                     isinstance(col.server_default, util.sqla_compat.Computed)
-                    and col.server_default.persisted
+                    and col.server_default and col.server_default.persisted
```
Comment (CaselIT) [inline alembic/ddl/sqlite.py:63]: Ok, this is because of the mock we do in sqla_compat.

Thanks for reporting, but the fix should be something else.

**Review by CaselIT — CHANGES_REQUESTED** [review]
I'll create an issue

File: alembic/ddl/sqlite.py (hunk 1 above)
Comment (hustliyilin) [reply alembic/ddl/sqlite.py:63]: OK. If you fix this issue, please let me know. Thanks.

**Comment by hustliyilin** [comment]
OK，Thanks.

**Pull request closed without merging**